python/python_fastapi_server/routers/edr.py

```diff
@@ -9,18 +9,21 @@
 """
```

Comment:
Can you please use the same linter as before? I think it should be black? Can do this in vscode by selecting the formatter for this project.